### PR TITLE
[FSTORE-1445] Merge of hsfs and hsml docs into hopsworks

### PR DESCRIPTION
**Pull request by aversey**
The next PR: https://github.com/logicalclocks/hopsworks-api/pull/223

The previous PR: https://github.com/logicalclocks/hopsworks-api/pull/226

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -0,0 +1,130 @@
+# Hopsworks Client
```
Comment (kennethmhc) [inline docs/index.md:1]: This file is not required as it is auto-generated. https://github.com/logicalclocks/hopsworks-api/blob/dev-merge/auto_doc.py#L209

**Review by kennethmhc — CHANGES_REQUESTED** [review]

File: mkdocs.yml [hunk 2]
```diff
@@ -34,9 +34,50 @@ nav:
...
+        - Deployment state: generated/model-serving/predictor_state_api.md
+        - Deployment state condition: generated/model-serving/predictor_state_condition_api.md
+        - Predictor: generated/model-serving/predictor_api.md
+        - Transformer: generated/model-serving/transformer_api.md
+        - Inference Logger: generated/model-serving/inference_logger_api.md
+        - Inference Batcher: generated/model-serving/inference_batcher_api.md
+        - Resources: generated/model-serving/resources_api.md
     # Added to allow navigation using the side drawer
     - Hopsworks API: https://docs.hopsworks.ai/hopsworks-api/latest/
-    - MLOps API: https://docs.hopsworks.ai/machine-learning-api/latest/
```
[the remark this answers is not in the document]
Comment (aversey) [reply mkdocs.yml:39]: Machine learning API docs is also merged into this repo, so there will be no separate docs for it.

File: docs/index.md (hunk 1 above)
Comment (aversey) [reply docs/index.md:1]: Yes, the same is true for CONTRIBUTIONS.md in docs. Should I add both of them to gitignore?

Comment (kennethmhc) [reply docs/index.md:1]: Sounds good

**Review by kennethmhc — APPROVED** [review]

**Pull request merged**